CONTRIBUTING.md

```diff
@@ -0,0 +1,240 @@
+6. Push Origin 
+    - Check what your origin is by Changing your directory to your repository and running: `git remote -v`
+        - This should display `origin https://github.com/$Username/PSKoans`
+    - Push 
+        - Run: `git push origin master` to push your local repo commits to your Github repo. 
+            - Running this command will pop up a dialog box to log into GitHub to authenticate your push. 
+7. Pull Request
+    - Go to Github and do a pull request from the PSKoans main page.
+8. Collaborate and have your change approved and merged into the main branch!
+
+#### VSCode
+1. Fork the Repository.
+    - Go to https://github.com/vexx32/PSKoans and click the Fork button at the top right hand side of the page.
+2. Setup.
+    - Download and Install [VSCode](https://desktop.github.com).
+    - Download and Install [Git](#powershell-git) (Follow the setup instructions from the git setup).
+3. Clone Repository.
+    - Follow the options from either [GitHub Desktop](#github-desktop) or [PowerShell Git](#powershell-git).
+4. Make Changes!
+    - Fix some bugs, write some code! Save your work!
+5. Commit.
+    - Hit the "Source Control" button on the left hand side of the VSCode window.
+    - Stage your changes.
+        - Hit the ➕ on the right side of the files you want to commit.
+    - Add a commit message in the "message" box.
+    - Hit the ✔️ above the files to commit them to the local repo. 
+6. Push Origin 
+    - In the lower left hand corner you should see 0⬇️1⬆️ 
+    - This will push your commits up to the master branch of your fork. 
+7. Pull Request
+    - Go to Github and do a pull request from the PSKoans main page.
+8. Collaborate and have your change approved and merged into the main branch!
+        
+
+Your submissions and contributions will be reviewed and processed by the PSKoans Code Owners. You will receive notifications in the GitHub tools as well as the email account used for GitHub.
+
+### Question Declaration in Koan files
+
+In PSKoans, each question requires the "Seeker of Enlightenment" to replace text which will allow the pester tests to pass successfully. 
+The following are a list of prompts that a Seeker might encounter.
+
+`__`= Answers that request int or double inputs (aka numeric). The standard will be **two underscores**.
+
+`'____'` = Answers that request string inputs. The standard will be a **single quote** followed by **four underscores** followed by a **single quote**.
+
+`"____"` = Answers that request string inputs which **also** include Variables (aka $Variables). The standard will be a **double quote** followed by **four underscores** followed by a **double quote**.
+
+`$____` = Answers that request variable inputs (aka $Variables). The standard will be a **dollar sign** followed by **four underscores**.
+
+`____` = Answers that don't request any of the above options. If this were a Switch Statement, this would be the Default case. The standard will be **four underscores**.
```

Comment:
This would mainly be used as placeholders for function calls themselves, really, so won't get used a lot but probably good to point out that it can be used for this.

Speaking of, I need to update `Get-Blank` to be able to be used more effectively this way...

Reply:
```suggestion
`____` = General-purpose usage. This could be a command name, a hashtable key name, or a parameter argument that expects a string. Any place a bare string is a valid answer, this should be used. Use **four underscores** for bare string blanks.
```